recipes/weeder/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+TGT="$PREFIX/share/weeder2"
+[ -d "$TGT" ] || mkdir -p "$TGT"
+[ -d "${PREFIX}/bin" ] || mkdir -p "${PREFIX}/bin"
+
+cd "${SRC_DIR}"
+g++ weeder2.cpp -o weeder2 -O3
+cp -r FreqFiles $TGT/
```

Comment:
Yes, reducing the conda package size is the aim. This example also works - the idea is just that you trigger the download post-installation or give hints how a user can get the datasets.